### PR TITLE
in_node_exporter_metrics: Add macOS into a table of supported metrics

### PR DESCRIPTION
**Pull request by cosmo0920**
This PR is corresponding to https://github.com/fluent/fluent-bit/pull/7947.

### CODE REVIEW

File: pipeline/inputs/node-exporter-metrics.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The initial release of Node Exporter Metrics contains a subset of collectors and
 
 **Important note:** Metrics collected with Node Exporter Metrics flow through a separate pipeline from logs and current filters do not operate on top of metrics.
 
-This plugin is currently only supported on Linux based operating systems\
+This plugin is currently supported on Linux based operating systems for the most part and some parts of collecting metrics are offering for macOS.
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:15]:
```suggestion
This plugin is supported on Linux-based operating systems for the most part with macOS offering a reduced subset of metrics.
The table below indicates which collector is supported on macOS.
```

File: pipeline/inputs/node-exporter-metrics.md [hunk 2]
```diff
@@ -58,20 +58,20 @@ The following table describes the available collectors as part of this plugin. A
...
-| stat      | Exposes various statistics from `/proc/stat`. This includes boot time, forks, and interruptions. | Linux  | v1.8    |
-| time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
-| uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
-| vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |
-| systemd collector | Exposes statistics from systemd.                                                           | Linux  | v2.1.3  |
+| Name      | Description                                                                                      | OS           | Version      |
+| --------- | ------------------------------------------------------------------------------------------------ | ------------ | ------------ |
+| cpu       | Exposes CPU statistics.                                                                          | Linux,macOS  | v1.8,v2.2    |
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:63]: Should this be 2.2 version @cosmo0920 ?

Comment (cosmo0920) [reply pipeline/inputs/node-exporter-metrics.md:64]: Yes. macOS metrics should be in v2.2 if we ship in the near future. This feature is not targeted for v2.1.

**Review by patrick-stephens — APPROVED** [review]

**Comment by patrick-stephens** [comment]
@cosmo0920 can you check if the new collectors need to be marked now?

**Comment by cosmo0920** [comment]
Sure. I'll handle about it.

**Comment by cosmo0920** [comment]
I checked the documentation but the all of documented list is described correctly. The description of filesystem metrics is absent on the table. So, I sent a PR to add it here: https://github.com/fluent/fluent-bit-docs/pull/1279

**Pull request merged**